src/pattern-library/components/patterns/hooks/UseClickAwayPage.tsx

```diff
@@ -0,0 +1,49 @@
+import Library from '../../Library';
+
+export default function UseClickAwayPage() {
+  return (
+    <Library.Page
+      title="useClickAway"
+      intro={
+        <p>
+          <code>useClickAway</code> is a hook that can be used to invoke a
+          callback when clicking outside of a container.
+        </p>
```

Comment:
I found this description and demo unclear compared to the JSDoc comment on the `useClickAway` hook. From looking at the demo you'd think that an `onClick` handler on the othermost div would be sufficient.